content/news/052/index.md

```diff
@@ -76,6 +76,49 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Tooling Updates
 
+### [Rusty Playdate]
+
+![The Playdate console](../051/playdate.png)  
+_The Playdate console_
+
+[Rusty Playdate] ([GitHub][Rusty Playdate], [Mastodon][Rusty Playdate Masto]) by [@boozook](https://github.com/boozook)
+is the large set of crates with bindings, toolset for the full cycle of creating games for the [Playdate handheld console][playdate].
+
+Big part of the Rusty Playdate project is the `cargo-playdate` tool ([Crates.io][cargo-playdate crates-io], [GitHub][cargo-playdate gh])
+that helps to build games for [Playdate] hardware or a simulator. It works as a cargo-plugin as well as standalone.
+The tool 
+- manages the compilation of your program,
+- builds assets for the crate and its dependencies,
+- generates a manifest,
+- and assembles it all into a bundle that runs on the device or a simulator.
+
+In this month `cargo-playdate` v0.5 has been [released][Rusty Playdate Release] and received massive refactoring, bugfixes and new features:
+- support of [cargo's auto-targets][cargo-target-auto-discovery] _(targets such as `bin` or `example` that aren't declared in the Cargo.toml)_
+- [target-specific package-info][pdb-pdxinfo-override] inheritance from the main package-info
+- `package.metadata.playdate.options` inheritance from the `workspace.metadata`
+- incremental builds now work as expected - fixed an old problem where the tool corrupts cargo's cache, which triggered full rebuild
+
+The register decoder in the [`pd-symbolize-crashlog`][playdate-symbolize-v0.2.0] was also updated. 
+It now it properly decodes all available registers such as
```

Comment:
> It now it

English is not native for me, but I'm not sure that it's correct.

```suggestion
So now it properly decodes all available registers such as
```

Reply:
Whoops, good catch!